### ISSUE TITLE
Error link between old deleted block and page

### ISSUE THREAD

**Issue by le-grimpeur** [sulu/sulu-standard#846]
| Q | A
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.6.14

#### Actual Behavior

When we work on sulu project with more than 2 peoples.
When a sulu page is configured and contributed with sulu block, when someone push a xml update who delete this block the sulu page is blocked with the error :

The block type ["old_block_name"] has not been registered. Known block types are: [all bock still existing]

#### Expected Behavior

The page with the changed block still working

#### Steps to Reproduce

Create a page with associete block and contribute this block. Delete the definition's block from the page's xml.
Try to access at this page. 

#### Possible Solutions

Maybe create a sulu comand who purge old link between page et old block.

Sorry for bad english

**Comment by alexander-schranz** [comment]
Hello @le-grimpeur,

currently we have no clean up command which deletes not needed properties from the DB or reset block types to default. At current state you need to write the a migration yourself you can use phpcrmigration or the `bin/adminconsole doctrine:phpcr:nodes:update` command e.g.:

```bash
bin/adminconsole doctrine:phpcr:nodes:update --query="SELECT * FROM [nt:unstructured] WHERE [jcr:mixinTypes] = 'sulu:page' --apply-closure="\$node->setProperty('i18n:<locale>-blocks-type#0', 'newtypetype');"
```

In the apply closure you can write normal PHP Code in your case you need to loop through the types be the -length attribute of the blocks. You need to call the same command with bin/websiteconsole for the live workspace.

**Comment by danrot** [comment]
That seems to be fixed in https://github.com/sulu/sulu/pull/3988, and that should be released in 1.6.19. Updating to this version should fix your issue.
